packages/gatsby/src/redux/actions/public.js

```diff
@@ -1252,6 +1252,19 @@ actions.createJobV2 = (job: JobV2, plugin: Plugin) => (dispatch, getState) => {
   return createJobV2FromInternalJob(internalJob)(dispatch, getState)
 }
 
+actions.addGatsbyImageSourceUrl = (sourceUrl: string) => {
+  return {
+    type: `PROCESS_GATSBY_IMAGE_SOURCE_URL`,
+    payload: { sourceUrl },
+  }
+}
+
+actions.clearGatsbyImageSourceUrls = () => {
```

Comment:
same with this one

Reply:
This one can probably be private but made sense to organize it with "public" as well since `addGatsbyImageSourceUrl` [has to be public](https://github.com/gatsbyjs/gatsby/pull/35427#discussion_r878074878)

Reply:
Done